### DISCUSSION TITLE
Tuto for screen Shader or render layer ?

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#169] [Q&A] [answered]
Hey ! Is there any tuto for the screen shader or render layer ? 
I would like not apply the render layer/screen shader  to a specific texture/object, is that possible ? 

Like in #94 

Thanks !

**Comment by pragma37** [comment, marked as answer]
> I would like not apply the render layer/screen shader to a specific texture/object, is that possible ?

You could use one of the material Custom Outputs as a mask.  Like this:
![imagen](https://user-images.githubusercontent.com/36610999/143042690-9c29d4d0-e7dc-40b9-b192-c8a518f3652b.png)
[post-process-mask.zip](https://github.com/bnpr/Malt/files/7588756/post-process-mask.zip)

**Reply by Airioo** [reply]
thanks, will try it !
